editor/editor_file_system.cpp

```diff
@@ -532,7 +532,7 @@ bool EditorFileSystem::_update_scan_actions() {
 				String full_path = ia.dir->get_file_path(idx);
 				if (_test_for_reimport(full_path, false)) {
 					//must reimport
-					reimports.push_back(full_path);
+					reimports = _get_dependencies(full_path);
```

Comment:
Doesn't this discard all the other paths which had previously been added to `reimports`?

Reply:
Yes, that's a problem ...

Reply:
With this it's work to and don't discard the other paths
![image](https://user-images.githubusercontent.com/45402513/87035191-422ce280-c1e9-11ea-9917-a5732f21c74d.png)